src/app/models/types/radioSchedule.ts

```diff
@@ -0,0 +1,9 @@
+export interface RadioScheduleData {
```

Comment:
We are now retrospectively applying newly defined typescript types here to our `jsx` - could be messy and end up forgotten about if we don't refactor soon